h5c/vic/src/vic-webapp/e2e/vch-create-wizard/1-basic.e2e-spec.ts

```diff
@@ -198,18 +198,12 @@ describe('VCH Create Wizard - Basic', () => {
     let vchFound = false;
     page.switchFrame(iframeTabs);
     page.waitForElementToBePresent(dataGridCell);
-    const deletedVch = new RegExp(namePrefix + specRunId);
-    element.all(by.css(dataGridCell)).each(function(element, index) {
-      element.isPresent().then(present => {
-        if (present) {
-          element.getText().then(function(text) {
-            if (deletedVch.test(text)) {
-              vchFound = true;
-            }
-          });
-        }
-      })
+    const vchClrDgActionXpath = `//clr-dg-action-overflow[contains(@class, '${namePrefix + specRunId}')]`;
```

Comment:
I'm seeing this timeout 
✓ should delete created vch
//clr-dg-action-overflow[contains(@class, 'virtual-container-host-866')] true
: : 
✗ should verify the created vch has been deleted
      - Failed: Wait timed out after 60092ms

Reply:
Also when it tries to discover if the vch is powered on (similar query) it fails with...

StaleElementReferenceError: stale element reference: element is not attached to the page document
  (Session info: chrome=63.0.3239.132)
  (Driver info: chromedriver=2.35.528157 (4429ca2590d6988c0745c24c8858745aaaec01ef),platform=Mac OS X 10.11.6 x86_64) not found